spec/models/report_spec.rb

```diff
@@ -0,0 +1,151 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Report, type: :model do
+  describe '#editable?' do
+    let(:user) { create(:user) }
+    let(:report) { create(:report, user:) }
+
+    context '自分の日報の場合' do
+      subject { report.editable?(user) }
+
+      it { is_expected.to eq true }
+    end
+
+    context '他人の日報の場合' do
+      subject { report.editable?(other_user) }
+
+      let(:other_user) { create(:other_user) }
+
+      it { is_expected.to eq false }
+    end
+  end
+
+  describe '#created_on' do
+    let(:user) { create(:user) }
+    let(:report) { create(:report, user:) }
+
+    it '日付を返すこと' do
+      travel_to('2023-08-20 17:32:46'.in_time_zone) do
+        expect(report.created_on).to eq '2023-08-20'.to_date
+      end
+    end
+  end
+
+  describe '#save_mentions' do
+    let(:user) { create(:user) }
+    let(:other_user) { create(:other_user) }
+    let(:other_report1) { create(:report, user: other_user, content: <<~CONTENT) }
```

Comment:
```suggestion
    let(:other_report_1) { create(:report, user: other_user, content: <<~CONTENT) }
```

好みかもしれませんが、連番にするときはアンダースコアで区切る派です。

Reply:
確かに見やすいと思ったのですが、修正するとRubocopで指摘を受けてしまうことがわかりました。このままとします 🙏 

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/VariableNumber

Reply:
おのれRubocop……。